controllers/gce/firewalls/firewalls.go

```diff
@@ -61,8 +61,12 @@ func (fr *FirewallRules) Sync(nodePorts []int64, nodeNames []string) error {
 	name := fr.namer.FrName(suffix)
 	rule, _ := fr.cloud.GetFirewall(name)
 	if rule == nil {
+		firewall, err := fr.createFirewallObject(suffix, "GCE L7 firewall rule", nodePorts, nodeNames)
+		if err != nil {
+			return err
+		}
```

Comment:
Should we just call this createFirewallObject once earlier in the function and re-use it here and below?